src/main/java/com/mide/gangsaeng/bannedword/BannedWordResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.List;
+
+import lombok.Getter;
+
+@Getter
+public class BannedWordResponse<T> {
```

Comment:
Why we need generic in this class?

Reply:
I just want to it has flexibility Initially, I used the string type in Response type, but I changed it to object type later.
So i used generic type in this class. Is it not necessary?

Reply:
Is there any case that T is not `BannedWord`?

Reply:
> Is there any case that T is not `BannedWord`?

No, maybe not in the future. I change it.